lib/src/room.dart

```diff
@@ -112,7 +112,7 @@ class KurentoRoom extends ChangeNotifier implements Room {
     // Asynchronously subscribe to the room-related topics.
     await _mq.subscribe(_participantEventTopic, MqttQos.atMostOnce, _handleParticipantEventMessage);
     await _mq.subscribe(_participantTopic, MqttQos.atMostOnce, _handleParticipantMessage);
-    await _mq.subscribe(_serviceRequestPresenceTopic, MqttQos.atMostOnce, _handleServiceRequestPresenceMessage);
+    await _mq.subscribe(_serviceRequestPresenceTopic, MqttQos.atLeastOnce, _handleServiceRequestPresenceMessage);
```

Comment:
It's important that we don't miss messages sent to the service request presence topic, since that's how Platform informs us of the service request state changes (`ASSIGNED` => `STARTED` => `END`).